docs/language/functions.rst

```diff
@@ -19,14 +19,31 @@ which are provided in the return statement, the values of the return variables a
 of the function is returned. It is still possible to explicitly return some values
 with a return statement.
 
-Functions which are declared ``public`` will be present in the ABI and are callable
-externally. If a function is declared ``private`` then it is not callable externally,
-but it can be called from within the contract. If a function is defined outside a
-contract, then it cannot have a visibility specifier (e.g. ``public``).
-
 Any DocComment before a function will be include in the ABI. Currently only Substrate
 supports documentation in the ABI.
 
+Function visibility
+___________________
+
+Solidity functions have a visibility specifier that restricts the scope in which they can be called.
```

Comment:
Very often we duplicate things that are already documented in the official Solidity docs. I'm aware that this is a whole other discussion, but I think we could at least link the official docs (https://docs.soliditylang.org/en/v0.8.20/contracts.html#function-visibility)